### PR TITLE
Feature/tag searching

### PR DESCRIPTION
**Pull request by mistikk**
### Issue number
#479

### CODE REVIEW

**Comment by ue** [comment]
@mistikk please let me know when you dont with that for testing with real devices

**Comment by mistikk** [comment]
I tested with android real devices @ue

File: src/screens/searchResult/screen/searchResultScreen.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <View
+              tabLabel={intl.formatMessage({
+                id: 'search.posts',
+              })}
+              style={styles.tabbarItem}
+            >
+              <Posts pageType="posts" tag={tag} />
+            </View>
+            {/* <View
+              tabLabel={intl.formatMessage({
```
[the remark this answers is not in the document]
Comment (mistikk) [reply src/screens/searchResult/screen/searchResultScreen.js]: it is not neccessary right now but we will use on part 2

[the remark this answers is not in the document]
Comment (ue) [reply src/screens/searchResult/screen/searchResultScreen.js]: cool cool

**Pull request merged**